### PR TITLE
Update and restructure

### PR DESCRIPTION
**Pull request by carstene1ns**
- automatic menu, easier navigaion
- About/Copyright page and footer
- fixed links
- gem updates
- recompressed images
- markdown all the way (mostly at least)
- simplified stuff

### CODE REVIEW

**Comment by carstene1ns** [comment]
After the last commit this is possible:
https://ci.easyrpg.org/job/website-pr/ws/output/index.html

**Review by Ghabry — APPROVED** [review]
Two minor things:

macOS brew install commands looks strange:

Screenshot_20220415_214433

Download page does not use the jenkins_link stuff yet.

File: lib/navigation.rb [hunk 1]
```diff
@@ -29,40 +20,47 @@ def link_to_upper_page(text, attributes = {})
   attributes[:class] = "upper"
   attributes[:title] = "Back to #{upper} page"
 
-  link_to(text + " " + upper + " " + text, target, attributes)
+  link_to("&orarr; " + upper, target, attributes)
```
Comment (fdelapena) [inline lib/navigation.rb:23]: Is there a reason to use HTML entity instead of the unicode symbol directly?

**Review by fdelapena — APPROVED** [review]

File: lib/navigation.rb (hunk 1 above)
Comment (carstene1ns) [reply lib/navigation.rb:23]: Just nostalgia 🎱

File: content/about/website.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+## Design
+
+- (Original Author) *FIXME*
+- carstene1ns (additions, modifications, scss port)
+
+## Techniques used
+
+- HTML5
+- CSS3
```
[the remark this answers is not in the document]
Comment (carstene1ns) [reply content/about/website.md:32]: I guess it does not have any use to list HTML and CSS explicitly then, as there are not much other possibilities. Will remove this section completely.

File: lib/navigation.rb (hunk 1 above)
Comment (Ghabry) [reply lib/navigation.rb]: Markdown will also replace (c) with copy sign btw

**Pull request merged**